lib/modules/orangered.js

```diff
@@ -171,6 +167,18 @@ const setupFaviconBadge = _.once(() => {
 	return favicon;
 });
 
+function setupNativeButtons() {
```

Comment:
Maybe replace this and two of the globals with something like

```js
const nativeButtons = _.once(() => {
    const inbox = document.querySelector('#header-bottom-right #mail');
    let mailCount = document.querySelector('#header-bottom-right .message-count');
    if (!mailCount && (module.options.updateCurrentTab.value || module.options.updateOtherTabs.value)) {
        mailCount = document.createElement('a');
        mailCount.style = 'display: inline-none;';
        $(inbox).after(mailCount);
    }
    return { inbox, mailCount };
});
```

Also `inline-none`?